pom.xml

```
@@ -530,6 +533,10 @@
                         <exclude>docs/demo/**/*.ipynb</exclude>
                         <exclude>docs/demo/**/*.zpln</exclude>
                         <exclude>**/src/main/resources/META-INF/services/*</exclude>
+                        <!-- Apache Rat excludes target folder for projects that are included by
+                        default. The reason we are explicitly excluding the target folder is because
+                        the tests-spark310+ project isn't included by default -->
```

Recommend making this comment not specific to the test-spark310+ project, as it's really applicable to any conditionally-included project.
```suggestion
                        default, but there are some projects that are conditionally included.  -->
```